publish.js

```diff
@@ -1,13 +1,19 @@
 import ghpages from 'gh-pages';
 
+let message = 'Publish latest changes from ';
+message += process.env.CIRCLE_SHA1 ?
+  'openfisca/openfisca.org@' + process.env.CIRCLE_SHA1 :
+  'openfisca.org contents';
```

Comment:
> ```suggestion
> const message = `Publish latest changes from ${process.env.CIRCLE_SHA1 ? `openfisca/openfisca.org@${process.env.CIRCLE_SHA1}` : 'openfisca.org contents'}`;
> ```

Reply:
I decided to go against template literals because this codebase has been developed, tested and deployed on old versions of Node and I didn't want to spend the time to check which version exactly brought template literals and which version was used in production, so I went with the safe old way.